spec/amp-var-substitutions.md

```diff
@@ -107,3 +138,12 @@ may make a request to something like `https://foo.com/pixel?host=example.com`.
 **PAGE_VIEW_ID**
 
 Contains a string that is intended to be random and likely to be unique per URL, user and day.
+
+**TIMESTAMP**
+
+Use the special string `TIMESTAMP` to add the current timestamp (Epoch time).
```

Comment:
done.

changed to:
Use the special string `TIMESTAMP` to add the current number of seconds that
have elapsed since 1970. (Epoch time)